### DISCUSSION TITLE
Welcome to sp-dev-modernization Discussions!

### DISCUSSION THREAD

**Discussion by jansenbe** [pnp/modernization#496] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by somaliasil** [comment]
Does the page conversion from classic to modern works for SP 2010 as well? 
Basically i want to confirm can sharepoint pnp module works for sp 2010 as well?
